llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -752,6 +752,60 @@ static bool removeSYCLKernelsConstRefArray(Module &M) {
   return true;
 }
 
+SmallVector<module_split::ModuleDesc, 2>
+handleESIMD(module_split::ModuleDesc &&MDesc, bool &Modified,
+            bool &SplitOccurred) {
+  // Do SYCL/ESIMD splitting. It happens always, as ESIMD and SYCL must
+  // undergo different set of LLVMIR passes. After this they are linked back
+  // together to form single module with disjoint SYCL and ESIMD call graphs
+  // unless -split-esimd option is specified. The graphs become disjoint
+  // when linked back because functions shared between graphs are cloned and
+  // renamed.
+  SmallVector<module_split::ModuleDesc, 2> Result = module_split::splitByESIMD(
```

Comment:
nit: can be pls rename these as Splitter (Result) and getESIMDSplitter (splitByESIMD) to be named in the same way as other splitter? Just a nit.

Thanks